manuscript/recipes/duplicati.md

```diff
@@ -0,0 +1,115 @@
+hero: Duplicati - Yet another option to backup your exciting stuff. It's good to have options.
+
+# Duplicati
+
+Intro
+
+![duplicati Screenshot](../images/duplicati.jpg)
+
+
+[Duplicati](https://www.duplicati.com/) is a free and open-source backup software to store encrypted backups online For Windows, macOS and Linux (our favorite, yay!).
+
+Similar to the other backup solution options in the Cookbook, you can use Duplicati to backup all our data-at-rest to a wide variety of locations, including, but not limited to:
+
+- FTP servers
+- SSH servers
+- WebDAV endpoints
+- Backblaze B2
+- Tardigrade
+- Microsoft OneDrive
+- Amazon S3
+- Google Drive
+- box.com
+- Mega
+- hubiC
+- many others
```

Comment:
Reduce to 3 bullets - maybe combine a few of the more popular options on a single line?

I.e.

- Generic endpoints (FTP, SSH, or WebDAV servers)
- Cloud storage providers (S3, B2, etc)
- Cloud services (OneDrive, Googel Drive, etc)

Reply:
Fixed